### ISSUE TITLE
Package list shows only 10 first packages after installing a package, resets to page 1

### ISSUE THREAD

**Issue by frischnetz** [modxbot/migrate#8180]
frischnetz created Redmine issue ID 8180

The package list always resets to page one after installing or upgrading a package, instead of staying where it was. Also: after installing or upgrading a package the package list does not respect the packages-per-page field and always shows only the first 10 entries. Only after using the reload-button it shows the full list.

This makes installing a long list of packages quite tedious.

**Comment by modxbot** [comment]
romain submitted:

Duplicate of #6522
